tests/trainer/test_fsdp.py

```diff
@@ -63,8 +76,72 @@ def test_fsdp_device_initialization(model: ComposerClassifier, mixed_precision:
             assert (torch.equal(weight_1, weight_2))
 
 
+@pytest.mark.parametrize(
+    'model,expected_param_inits',
+    [
+        (SimpleModel, 2),  # One call for each of the Linear layers
+        (EmbeddedWeightTiedModel, 3),  # Two calls for each of the SimpleMLP modules, minus one for weight tying
+    ])
+@pytest.mark.parametrize('device', _INIT_DEVICES)
+@world_size(2)
+@pytest.mark.gpu
+@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('1.13.0'),
```

Comment:
Should this be skipped if we're using anything less than torch 2.1.0? I'd expect it to fail on earlier versions.
```suggestion
@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('2.1.0'),
```

Reply:
I _think_ this test should still pass on other torch versions.

Reply:
thinking about making the change apply to torch 2.0.1 as well

Reply:
alright, yeah, here is when it changed: https://github.com/pytorch/pytorch/pull/104189#discussion_r1248269410